src/custom_nodes/model_zoo_intel_object_detection/model_zoo_intel_object_detection.cpp

```diff
@@ -404,11 +408,9 @@ int execute(const struct CustomNodeTensor* inputs, int inputsCount, struct Custo
     CustomNodeLibraryInternalManager* internalManager = static_cast<CustomNodeLibraryInternalManager*>(customNodeLibraryInternalManager);
 
     *outputsCount = 3;
-    struct CustomNodeTensor* out = nullptr;
-    if (!get_preallocated_buffer<struct CustomNodeTensor>(internalManager, &out, OUTPUT_TENSOR_NAME, 3 * sizeof(CustomNodeTensor))) {
+    if (!get_preallocated_buffer<struct CustomNodeTensor>(internalManager, &(*outputs), OUTPUT_TENSOR_NAME, 3 * sizeof(CustomNodeTensor))) {
```

Comment:
You dereference to get the value (`*outputs`) from `**outputs` and then you take the pointer (`&(*outputs)`) to get `**outputs` back. This operation does nothing, you can just pass `outputs`:
```
if (!get_preallocated_buffer<struct CustomNodeTensor>(internalManager, outputs, OUTPUT_TENSOR_NAME, 3 * sizeof(CustomNodeTensor))) {
```